client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -30,6 +30,7 @@ export class CommentDetailAuthor extends Component {
 		authorUsername: PropTypes.string,
 		blockUser: PropTypes.func,
 		commentDate: PropTypes.string,
+		commentIsPing: PropTypes.bool,
```

Comment:
Could we pass through commentType instead, it'd be an enum of 'comment', 'ping', and could be expanded later if say reactions land 😆 

Since we also have `showAuthorInfo` I would recommend changing the value from the caller, instead of adding another case in the component. If I pass though`showAuthorInfo=true` and `commentIsPing=true` without looking at the internal details, I'd expect to see the author info.

Reply:
We might also consider not passing through `commentIsPing` or `commentType` and maybe pass dumb boolean props instead like `showLinkIcon`